pilot/pkg/model/authentication.go

```diff
@@ -47,6 +51,31 @@ func GetConsolidateAuthenticationPolicy(mesh *meshconfig.MeshConfig, store Istio
 	return legacyAuthenticationPolicyToPolicy(mesh.AuthPolicy)
 }
 
+// ConstructSdsSecretConfig constructs SDS Sececret Configuration.
+func ConstructSdsSecretConfig(serviceAccount string, meshConfig *meshconfig.MeshConfig) *auth.SdsSecretConfig {
+	refreshDuration, _ := ptypes.Duration(meshConfig.SdsRefreshDelay)
+	return &auth.SdsSecretConfig{
+		Name: serviceAccount,
```

Comment:
Does the name play any role in choosing Sds config to use? I might be wrong, but I feel it would be ok to have just one Sds config, with a hard coded name, as we don't have different SDS service (i.e SdsUdsPath) yet.

Reply:
I see. Thanks.